pfcpiface/conn.go

```diff
@@ -121,6 +140,13 @@ func (node *PFCPNode) NewPFCPConn(lAddr, rAddr string, buf []byte) *PFCPConn {
 	log.Infoln("Created PFCPConn from:", conn.LocalAddr(), "to:", conn.RemoteAddr())
 
 	rng := rand.New(rand.NewSource(time.Now().UnixNano())) // #nosec G404
+
+	pConnId := node.allocatePFCPConnId()
+	if pConnId == 0 {
```

Comment:
why would 0 be an invalid value? I suggest returning `(pConnId, error)` from `allocatePFCPConnId()` and check error.